_scripts/app2-3.R

```diff
@@ -0,0 +1,53 @@
+{
+  
+  # Application 2-3 ----
+  
+  rm(list=ls());
+  options(show.error.locations = TRUE);
+  
+  # Part A ----
+  
+  vector1 <- seq(from = 45, to = 167, by = 1); # alt. = seq(45:167)
+  vector2 <- seq(from = 25, to = -7, by = -1); # alt. = seq(25:-7)
+  
+  # Part B ----
+  
+  vector3 <- seq(from = 17, to = 5, by = -3);
+  
+  for(j in vector3){
```

Comment:
An alternate way to do this for()

for(j in 1:length(vector3))  # so j = 1,2,3,4,5 because there are 5 values in vector3
{
   # j is now an index value for vector3 as opposed to being the values in vector3
    cat("\n", vector3[j], "cubed power is", vector3[j]^3, "\n");
    cat("\n", vector3[j], "cubed root is", vector3[j]^(1/3), "\n");
}